src/components/ChecSlideoutPanel/Panel.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <transition
+    name="slide"
+    mode="out-in"
+  >
+    <div
+      class="slideout-panel__element"
+      :class="classNames"
+    >
+      <div class="slideout-panel__header">
+        <component :is="titleTag" v-if="title" class="slideout-panel__title">
+          {{ title }}
+        </component>
+        <ChecButton
+          class="slide-panel__close"
+          color="secondary"
+          variant="small"
+          @click="emitClose"
+        >
+          <template #icon>
+            <ChecIcon icon="close" />
+          </template>
+        </ChecButton>
+      </div>
+      <div class="slideout-panel__content" :class="`max-w-${width}`">
+        <!--
+          @slot Panel content
+        -->
+        <slot />
+      </div>
+      <div v-if="$slots.toolbar" class="slideout-panel__footer">
+        <!--
+          @slot Toolbar actions, buttons, etc
+        -->
+        <slot name="toolbar" />
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import ChecButton from '../ChecButton.vue';
+import ChecIcon from '../ChecIcon';
+
+export default {
+  name: 'Panel',
+  components: {
+    ChecButton,
+    ChecIcon,
+  },
+  props: {
+    /**
+     * The size of the panel width. One of "half", "third", "threeQuarters", "full".
+     */
+    size: {
+      type: String,
+      default: 'half',
```

Comment:
I asked about using Tailwind widths in the other component - if you change it there then please do it here too. If not then ignore this.